docs/advanced-features/preview-mode.md

```diff
@@ -221,6 +221,10 @@ export default function myApiRoute(req, res) {
 Both the bypass cookie value and the private key for encrypting the `previewData` change when `next build` is completed.
 This ensures that the bypass cookie can’t be guessed.
 
+### Developing locally
+
+To test Preview Mode locally over HTTP, ensure the `NODE_ENV` environment variable is set to `development`. Your browser will need to allow third-party cookies and local storage access.
```

Comment:
when using `next dev` it's automatically set to `development`. I think we can change this to a note instead.

```suggestion
> **Note:** To test Preview Mode locally over HTTP your browser will need to allow third-party cookies and local storage access.
```

Reply:
How are people getting into a state where development isn't set?

Reply:
Sorry for raising the hassle. I wasn't aware this was set with `next dev`. We've got a custom `dev` script (`"dev": "yarn cf-ts-gen && node cacheBuster.js && node server.js"`).

Reply:
Ah, okay. That makes sense!

Reply:
@lfades should we add a note about `next dev` here then?

Reply:
I'm not sure if we have issues with de behavior of `NODE_ENV` in `next dev` / `next start` currently. We set the value because it shouldn't be anything else and we don't recommend Next.js using a different method. But we can mention it on docs if that's confusing.

Reply:
The reason why I created this PR was that in my case `NODE_ENV` wasn't set at all, and I wasn't aware Next.js was relying on it. As most developers will use `next dev` this isn't a huge problem. 

Some more context: I took over somebody else's codebase with a custom `yarn dev` and I totally didn't realize `NODE_ENV` should have been set, until I couldn't get the Preview Mode working.

Reply:
@ludder I think it's okay to make the update to the custom server docs instead 🤔